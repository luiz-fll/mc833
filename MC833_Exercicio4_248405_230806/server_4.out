remoto: 127.0.0.1:6004
[SERVIDOR] Escutando em 127.0.0.1:6004
127.0.0.1:45974
GET / HTTP/1.0
Host: teste

127.0.0.1:45984
GET / HTTP/1.0
Host: teste

127.0.0.1:45992
GET / HTTP/1.0
Host: teste

127.0.0.1:46008
GET / HTTP/1.0
Host: teste

127.0.0.1:46016
GET / HTTP/1.0
Host: teste

127.0.0.1:46018
GET / HTTP/1.0
Host: teste

127.0.0.1:46028
GET / HTTP/1.0
Host: teste

127.0.0.1:46040
GET / HTTP/1.0
Host: teste

127.0.0.1:46056
GET / HTTP/1.0
Host: teste

