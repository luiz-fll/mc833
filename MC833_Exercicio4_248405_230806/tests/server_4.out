remoto: 127.0.0.1:60004
[SERVIDOR] Escutando em 127.0.0.1:60004
127.0.0.1:55036
GET / HTTP/1.0
Host: teste

127.0.0.1:55042
GET / HTTP/1.0
Host: teste

127.0.0.1:55054
GET / HTTP/1.0
Host: teste

127.0.0.1:55064
GET / HTTP/1.0
Host: teste

127.0.0.1:55070
GET / HTTP/1.0
Host: teste

127.0.0.1:55086
GET / HTTP/1.0
Host: teste

127.0.0.1:55092
GET / HTTP/1.0
Host: teste

127.0.0.1:55098
GET / HTTP/1.0
Host: teste

127.0.0.1:55104
GET / HTTP/1.0
Host: teste

